tech/filters/component-render.ts

```diff
@@ -0,0 +1,12 @@
+import { docUtils } from './doc-utils';
+import { Options, Render } from './Documentation';
+
+const render = (height: string): Render => (src) => `<iframe class="doc-iframe" src="${src}" height="${height}"></iframe>`;
+
+export const componentRender = (str: string, opts: Options): string => {
```

Comment:
What is the purpose of render only ? 

Do we need to keep this filter ?

Reply:
I don't understand, can you be more explicit, maybe by using an example ?

Reply:
View with @Gnuk, we close this conversation